index.html

```diff
@@ -4,16 +4,48 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width" />
     <title>Awesome Books</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="./styles.css">
   </head>
-
   <body>
-    <h1>Awesome Books</h1>
-    <div id="books"></div>
-    <form id="form">
-      <input id="title" type="text" placeholder="Title" />
-      <input id="author" type="text" placeholder="Author" />
-      <button id="add-button" type="submit">Add</button>
-    </form>
+    <header class ="header">
+      <nav class="container-l navbar navbar-expand-lg navbar-light bg-light p-3">
+        <h1 class="col-lg-9 h3">Awesome Books</h1>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse mr-auto col-lg-3 justify-content-around" id="navbarText">
+          <ul class="navbar-nav">
+            <li class="nav-item active">
+              <a class="nav-link" href="./index.html">List</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./add-new.html">Add New</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./contact.html">Contact</a>
```

Comment:
- [ ] You should not have different pages for these sections. If you check the project requirement, you can see that you need to manipulate the content of the page with javascript in order the change sections. (When I click on the link only the content of the main area should change, not the page.)
You can simply add (classList.add) and remove (classList.remove) CSS classes to display or hide the content.

Reply:
Hi sinansevgi, ok perfect you are totally right, Now we fix the code to manipulate the content of the page with javascript in order the change sections :smile: .

Reply:
This issue is well resolved! ✔️